protocol-specs/generators/block_processing_multisignatures/index.js

```diff
@@ -0,0 +1,401 @@
+		.to('1670991471799963578L')
+		.transfer('30')
+		.from('10881167371402274308L')
+		.to('2222471382442610527L')
+		.forge();
+
+	// Register multisignature and two co-signers for it
+	chainStateBuilder
+		.registerMultisignature('2222471382442610527L')
+		.addMemberAndSign('8465920867403822059L')
+		.addMemberAndSign('1670991471799963578L')
+		.finish();
+	// Tranfer from the new multisignature account
+	chainStateBuilder
+		.transfer('7')
+		.from('2222471382442610527L')
+		.to('10881167371402274308L');
+
+	chainStateBuilder
+		.signTransaction(chainStateBuilder.lastTransactionId)
+		.withAccount('8465920867403822059L');
+
+	chainStateBuilder
+		.signTransaction(chainStateBuilder.lastTransactionId)
+		.withAccount('1670991471799963578L')
+		.forgeInvalidInputBlock();
+
+	const chainAndAccountStates = chainStateBuilder.getScenario();
+
+	return {
+		initialState: {
+			// Given the library chainStateBuilder saves all mutations we use slice here to pick the first accounts state
+			chain: chainAndAccountStates.chain.slice(0, 2),
+			accounts: chainAndAccountStates.finalAccountsState[4],
+		},
+		input: chainAndAccountStates.inputBlock,
+		output: {
+			chain: chainAndAccountStates.chain,
+			// Given the library chainStateBuilder saves all mutations we use slice here to pick the last account state
+			accounts: chainAndAccountStates.finalAccountsState.slice(-1),
+		},
+	};
+};
+
+const validBlockWithMultisignatureRegistrationTx = () => ({
+	title: 'Valid block processing',
+	summary:
+		'A valid block with a multisignature registration transaction processed',
+	config: 'mainnet',
+	runner: 'block_processing_multisignatures',
+	handler: 'valid_block_processing_multisignature_registration_tx',
+	testCases: generateTestCasesValidBlockMultisignatureRegistrationTx(),
+});
+
+const validBlockWithTransferFromMultisigAccount = () => ({
+	title: 'Valid block processing',
+	summary:
+		'A valid block with a transfer transaction from a multisignature account processed',
+	config: 'mainnet',
+	runner: 'block_processing_multisignatures',
+	handler: 'valid_block_processing_transfer_from_multisignature_account',
+	testCases: generateTestCasesValidBlockTransferFromMultisignatureAccount(),
+});
+
+const invalidBlockWithMultisignatureRegistrationAndFundingInSameBlock = () => ({
+	title: 'Invalid block processing',
+	summary:
+		'An invalid block with a multisignature registration transaction and funding for members in same block',
+	config: 'mainnet',
+	runner: 'block_processing_multisignatures',
+	handler:
+		'invalid_block_processing_multisignature_registration_and_funding_for_members_same_block',
+	testCases: generateTestCasesInvalidBlockMultisignatureRegistrationAndFundingInSameBlock(),
+});
+
+const invalidBlockWithTransferFromMultisigAccountOnSameBlockAsRegistration = () => ({
+	title: 'Invalid block processing',
+	summary:
+		'An invalid block with a multisignature registration and transfer from that account on same block',
+	config: 'mainnet',
```

Comment:
I just realized but why we use `mainnet` config?

Reply:
I was just using it from the previous implementation. I don't know if that value is currently being used at all.